docs/Blog-posts.md

```diff
@@ -3,6 +3,6 @@
  - [Trying out "dotnet new" template updates and csproj with VS2017](https://www.hanselman.com/blog/TryingOutDotnetNewTemplateUpdatesAndCsprojWithVS2017.aspx) by @shanselman
  - [Custom Project Templates Using dotnet new](http://rehansaeed.com/custom-project-templates-using-dotnet-new/) by @RehanSaeed
  - [dotnet new Feature Selection](http://rehansaeed.com/dotnet-new-feature-selection/) by @RehanSaeed
- - [Building a Solution with dotnet cli nuget templates](https://www.motowilliams.com/building-a-solution-with-dotnet-cli-nuget-templates) by @motowilliams
+ - [Building a Solution with dotnet cli nuget templates](https://motowilliams.com/2017-02-21-building-a-solution-with-dotnet-cli-nuget-templates/) by @motowilliams
 - [Building And Debugging The dotnet new Templating Engine In Visual Studio](http://pioneercode.com/post/building-and-debugging-the-dot-net-new-templating-engine-in-visual-studio) by @chad-ramos
 - [How To Create A dotnet new Project Template In .NET Core](http://pioneercode.com/post/how-to-create-a-dot-net-new-project-template-in-dot-net-core) - @chad-ramos
```

Comment:
The [pioneercode.com](https://pioneercode.com) is currently unavailable as well. Trying to see if that's just a temporary outage or if we should remove those last two as well